backend/app/Http/Requests/UserUpdatePasswordRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UserUpdatePasswordRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
```

Comment:
same here for adding checks

Reply:
Should we allow the admin to update the user password? I believe admin won't be able to update the password of the user since they don't know the old_password of the user.

Reply:
Will add logic instead to allow if profile is the current logged user.

Reply:
@jescabillas ah right, lets authorize lang same user